render/gles2/shaders.c

```diff
@@ -6,14 +6,14 @@ const GLchar quad_vertex_src[] =
 "uniform mat3 proj;\n"
 "uniform vec4 color;\n"
 "attribute vec2 pos;\n"
-"attribute vec2 texcoord;\n"
+//"attribute vec2 texcoord;\n"
```

Comment:
We can probably get rid of this (and the rest of the texcoord stuff) for colored quads

Reply:
Oops, this was from a debug session. But yes, the `pos` vertices and the `texcoord` vertices are the same, I think we could reuse the same array of vertices.

Reply:
This has been put back.